pythonforandroid/recipes/matplotlib/mpl_android_fixes.patch

```diff
@@ -1,29 +1,64 @@
-diff --git a/setupext.py b/setupext.py
-index fc82d5d..2067db0 100644
---- a/setupext.py
-+++ b/setupext.py
-@@ -1004,10 +1004,10 @@ class Numpy(SetupPackage):
-         ext.define_macros.append(('__STDC_FORMAT_MACROS', 1))
-
+--- matplotlib-3.1.1/setupext.py.orig	2019-12-31 01:25:00.000000000 +0100
++++ matplotlib-3.1.1/setupext.py	2020-01-04 15:30:50.453260814 +0100
+@@ -604,8 +604,7 @@ class Numpy(SetupPackage):
+     name = "numpy"
+ 
+     def add_flags(self, ext):
+-        import numpy as np
+-        ext.include_dirs.append(np.get_include())
++        ext.include_dirs.append(os.environ['NUMPY_INCLUDES'])
+         ext.define_macros.extend([
+             # Ensure that PY_ARRAY_UNIQUE_SYMBOL is uniquely defined for each
+             # extension.
+@@ -617,10 +616,10 @@ class Numpy(SetupPackage):
+         ])
+ 
      def get_setup_requires(self):
--        return ['numpy>=1.10.0']
-+        return ['numpy==1.16.4']  # to match p4a's target version
-
+-        return ['numpy>=1.11']
++        return []  # don't need it for p4a, due to above changes
+ 
      def get_install_requires(self):
--        return ['numpy>=1.10.0']
+-        return ['numpy>=1.11']
 +        return ['numpy==1.16.4']  # to match p4a's target version
```

Comment:
I have some doubts with this line. Does it really have to be hard pinned to the current default p4a numpy version?
And if it does, should we at some point try to pick this up dynamically?

Reply:
> I have some doubts with this line. Does it really have to be hard pinned to the current default p4a numpy version?

I'm not sure, I'm doing a local test...we will see

> And if it does, should we at some point try to pick this up dynamically?

I think so, but it will add more complexity to the recipe...

Reply:
Yes I was also thinking about the complexity. I think we can address that later if needed.
You have my :heavy_check_mark: feel free to merge if it builds :green_heart: